docs/roadmap.md

```diff
@@ -0,0 +1,36 @@
+# `rust-bitcoin` repository roadmap
+
+* ~Release `units v1.0.0-rc0`~
```

Comment:
In a3c03adf407483240417df158db7a6230e9cb45e:

This `rc` syntax should have another `.` in it to match the versions that we're putting on crates.io.

Reply:
Woops, done. Thanks